libraries/botframework-schema/tests/activityEx.test.js

```diff
@@ -0,0 +1,577 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const assert = require('assert');

```

Comment:
```suggestion
const { strictEqual } = require('assert');
```

And remove `assert.` occurrences